apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking/useDeepLinkHandler.ts

```diff
@@ -203,9 +203,26 @@ export function useDeepLinkHandler() {
           }
           break;
         }
-        case "swap":
-          navigate("/swap");
+        case "swap": {
+          const { amountFrom, fromToken, toToken } = query;
+
+          const state: {
```

Comment:
Should there be a type for this, do you think?

I don't know whether `state` can just be arbitrary keys and values or is something with more restricted scope, so genuine question

Reply:
the state is just eventually passed into history.push which by default takes an `unknown` type, since under the hood it uses the History API where the state can be "anything that can be serialized"